common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -296,6 +310,20 @@ private void openLinkInBrowser(final String url) {
         }
     }
 
+    private boolean processWebCpRequest(@NonNull final WebView view, @NonNull final String url) {
+
+        view.stopLoading();
+        
+        if (url.equalsIgnoreCase(AuthenticationConstants.Broker.WEBCP_LAUNCH_COMPANY_PORTAL_URL)) {
```

Comment:
do we want to be case-insensitive here?

Reply:
I'm inclined toward a 'no'.

While we don't really have an agreement on case-sensitivity (it's more like "I'll give you X and you do Y"), I don't think client side should break if the server side accidentally ship an upper case bug (we've seen that before).